src/main/java/uk/gov/ons/ctp/common/util/WebDriverUtils.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.ons.ctp.common.util;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxDriverLogLevel;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.logging.LogType;
+import org.openqa.selenium.logging.LoggingPreferences;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class WebDriverUtils {
+
+  public static WebDriver getWebDriver(
+      final WebDriverType webDriverType, final boolean isHeadless, final Level loggingLevel) {
+    LoggingPreferences logs = new LoggingPreferences();
+    logs.enable(LogType.BROWSER, loggingLevel);
+    logs.enable(LogType.CLIENT, loggingLevel);
+    logs.enable(LogType.DRIVER, loggingLevel);
+    logs.enable(LogType.PERFORMANCE, loggingLevel);
+    logs.enable(LogType.PROFILER, loggingLevel);
+    logs.enable(LogType.SERVER, loggingLevel);
+
+    DesiredCapabilities desiredCapabilities = DesiredCapabilities.firefox();
```

Comment:
I think this is the part that failed in dev the first time. The desiredCapabilities are being set for firefox but are then being used by all the different types of driver (including Chrome).